support/rollup/examples.js

```diff
@@ -0,0 +1,66 @@
+import path from 'path'
+import alias from 'rollup-plugin-alias'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import json from 'rollup-plugin-json'
+import replace from 'rollup-plugin-replace'
+import babel from 'rollup-plugin-babel'
+import uglify from 'rollup-plugin-uglify'
+import sourcemaps from 'rollup-plugin-sourcemaps'
+import pkg from '../../package.json'
+
+const configurations = []
+
+const umdConfig = {
+  input: 'examples/index.js',
+  output: {
+    file: pkg.browser,
+    name: 'slate-examples',
+    format: 'umd',
+    exports: 'named',
+    sourcemap: process.env.ROLLUP_WATCH,
```

Comment:
Can you collect all `env` variables near the top of the file somewhere, so we can reference them just as `ROLLUP_WATCH`, etc. from here on out?

Should this be `ROLLUP_WATCH`, or should it actually be `NODE_ENV != 'production'`? I think using Rollup's watch feels slightly incorrect?

Reply:
Yeah, kinda six one/half a dozen the other (because we only build non-production when also watching), but I agree yours is probably a little more clear